tmp_estimator_name

```diff
@@ -0,0 +1 @@
+DNNClassifier
```

Comment:
Is this file a leftover for debugging?

Reply:
This is a file fetched from OSS when predicting a model, should add this in `.gitignore`